bitnami/postgresql/README.md

```diff
@@ -210,7 +210,7 @@ $ kubectl delete pvc -l release=my-release
 | `primary.priorityClassName`                   | Priority Class to use for each pod (postgresql primary)                                                                                                   | `""`                        |
 | `primary.extraInitContainers`                 | Extra init containers to add to the pods (postgresql primary)                                                                                             | `[]`                        |
 | `primary.extraVolumeMounts`                   | Extra volume mounts to add to the pods (postgresql primary)                                                                                               | `[]`                        |
-| `primary.extraVolumes`                        | Extra volumes to add to the pods (postgresql primary)                                                                                                     | `[]`                        |
+| `primary.extraVolumes`                        | Extra volumes to add to the pods (postgresql primary), the value is evaluated as a template.                                                                                                    | `[]`                        |
```

Comment:
I think that you don't need to include this in the Readme. I took a look another charts like Redis and we are not including this information. So, please remove it. Thanks.